all-the-icons.el

```diff
@@ -390,33 +390,101 @@
-    (benchmark-init/tree-mode  all-the-icons-octicon "dashboard"           :v-adjust 0.0)
-    (jenkins-mode              all-the-icons-fileicon "jenkins")
-    (magit-popup-mode          all-the-icons-alltheicon "git")
-    (magit-status-mode         all-the-icons-alltheicon "git")
+    (help-mode                 all-the-icons-faicon "info"                 :v-adjust -0.1 :face all-the-icons-purple)
+    (benchmark-init/tree-mode  all-the-icons-octicon "dashboard"           :v-adjust 0.0//)
+    (jenkins-mode              all-the-icons-fileicon "jenkins"            :face all-the-icons-blue)
+    (magit-popup-mode          all-the-icons-alltheicon "git"              :face all-the-icons-red)
+    (magit-status-mode         all-the-icons-alltheicon "git"              :face all-the-icons-lred)
+    (magit-log-mode            all-the-icons-alltheicon "git"              :face all-the-icons-green)
 
     ;; Special matcher for Web Mode based on the `web-mode-content-type' of the current buffer
-    (web-mode                 all-the-icons--web-mode-icon)
+    (web-mode             all-the-icons--web-mode-icon)
+
+    (fundamental-mode                   all-the-icons-fileicon "elisp"            :height 1.0 :v-adjust -0.2 :face all-the-icons-dsilver)
+    (special-mode                       all-the-icons-fileicon "elisp"            :height 1.0 :v-adjust -0.2 :face all-the-icons-yellow)
+    (text-mode                          all-the-icons-octicon "file-text"         :v-adjust 0.0 :face all-the-icons-cyan)
+    (ruby-mode                          all-the-icons-alltheicon "ruby-alt"       :face all-the-icons-lred)
+    (inf-ruby-mode                      all-the-icons-alltheicon "ruby-alt"       :face all-the-icons-red)
+    (projectile-rails-compilation-mode  all-the-icons-alltheicon "ruby-alt"       :face all-the-icons-red)
+    (rspec-compilation-mode             all-the-icons-alltheicon "ruby-alt"       :face all-the-icons-red)
+    (rake-compilation-mode              all-the-icons-alltheicon "ruby-alt"       :face all-the-icons-red)
+    (shell-mode                         all-the-icons-alltheicon "terminal"       :face all-the-icons-purple)
+    (fish-mode                          all-the-icons-alltheicon "terminal"       :face all-the-icons-lpink)
+    (nginx-mode                         all-the-icons-fileicon "nginx"            :height 0.9  :face all-the-icons-dgreen)
+    (apache-mode                        all-the-icons-alltheicon "apache"         :height 0.9  :face all-the-icons-dgreen)
+    (makefile-mode                      all-the-icons-fileicon "gnu"              :face all-the-icons-dorange)
+    (dockerfile-mode                    all-the-icons-fileicon "gnu"              :face all-the-icons-dorange)
+    (xml-mode                           all-the-icons-faicon "file-code-o"        :height 0.95 :face all-the-icons-lorange)
+    (json-mode                          all-the-icons-alltheicon "aws"            :face all-the-icons-orange)
```

Comment:
`json` doesn't make sense to use the `aws` icon... I think it should be `settings` like `yaml` below

(see [this line](https://github.com/asok/all-the-icons.el/blob/extend-buffer-icons-alist/all-the-icons.el#L172))